projects/packages/wp-admin-plus/package.json

```diff
@@ -0,0 +1,39 @@
+{
+	"name": "jetpack-wpadmin-plus-ui",
+	"version": "0.0.1-alpha",
+	"description": "WPAdmin Plus UI",
+	"main": "_inc/index.js",
+	"repository": "https://github.com/automattic/jetpack-wp-admin-plus",
+	"author": "Automattic",
+	"license": "GPL-2.0-or-later",
+	"scripts": {
+		"install-if-deps-outdated": "pnpm install --no-prod --frozen-lockfile",
+		"build-all": "pnpm run install-if-deps-outdated && pnpm run clean && pnpm run build",
+		"build": "wp-scripts build src/index.js",
+		"start": "wp-scripts start src/index.js",
+		"clean": "rm -rf build/",
+		"lint:css": "wp-scripts lint-style",
+		"lint:js": "wp-scripts lint-js",
+		"watch": "pnpm run build-all && pnpm run start"
+	},
+	"dependencies": {
+		"@wordpress/components": "15.0.0",
+		"@wordpress/core-data": "4.0.0",
+		"@wordpress/data": "5.2.0",
+		"@wordpress/date": "4.2.1",
+		"@wordpress/dom-ready": "3.2.1",
+		"@wordpress/editor": "10.2.0",
+		"@wordpress/element": "4.0.0",
+		"@wordpress/i18n": "4.2.1"
+	},
+	"devDependencies": {
```

Comment:
I had to add "@babel/runtime": "7.14.0", to get this package working on my dev environment. It'd be worth getting someone else to install this package to see if they have the same issue.

Reply:
I had the same issue, and the same fix worked. 👍

Reply:
mmm, have tried updating `pnpm`?

Reply:
`pnpm install -f` also fails, with `gyp: No Xcode or CLT version detected!`. I could try reinstalling Xcode.

Reply:
I reinstalled Xcode and ran `pnpm install -f` successfully, but I still get the `Module not found: Error: Can't resolve '@babel/runtime/helpers/extends'` error.

Reply:
:-/ let me add the package into the dependencies